tests/integration/message_queue_tests.py

```diff
@@ -4,11 +4,12 @@
 
 import posix_ipc
 
-from baseplate.lib.message_queue import MessageQueue
+from baseplate.lib.message_queue import InMemoryMessageQueue
```

Comment:
I think you should add an IT for the remote queue. If you make the port configurable you should be able to make sure it is unique on the test machine and run a test over localhost using the remote queue thrift service and `RemoteQueueRequestHandler`.